chsdi/views/transports.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+from sqlalchemy import and_
+from pyramid.view import view_config, view_defaults
+from chsdi.models.vector.uvek import Oev_departures
+import datetime
+
+
+@view_defaults(renderer='jsonp', route_name='transports')
+class TransportView(object):
+
+    def __init__(self, request):
+        self.request = request
+        if request.matched_route.name == 'transports':
```

Comment:
This condition is redundant with the route_name in the decorator above... So this will always be true.